packages/core-library/tests/page/hub/inclusion/InclusionForm.test.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { screen, render, fireEvent, waitFor } from "../../../common";
+import { InclusionForm } from '../../../../system/app/internal/blocks/Hub/Product/inclusion/InclusionForm';
+import { act } from 'react-dom/test-utils';
```

Comment:
moved this to common
```suggestion
import { act, screen, render, fireEvent, waitFor } from "../../../common";
```